package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "js-component-framework",
+  "version": "1.0.0",
+  "description": "This framework is a method of attaching an ES6 class to a DOM element or collection of DOM elements.",
+  "main": "lib/index.js",
+  "scripts": {
+    "prepare": "./node_modules/.bin/babel src --out-dir lib",
```

Comment:
I'd recommend including only the `lib` directory in the `npm` package via the `files` property in this `package.json`: https://docs.npmjs.com/files/package.json#files